### DISCUSSION TITLE
How to delete a solution?

### DISCUSSION THREAD

**Discussion by M4N0JKUM4R** [cssbattle/forum#50] [Q&A] [answered]
Is there a feature to delete a solution? I remember making the account 2 years back and used one of the top solutions to test the platform as it has the "Load code" button. But now my own solution needs to be taken into account as the highest score instead of the other top solution by the user of that time.

![image](https://user-images.githubusercontent.com/11471878/114401177-4d6dce00-9bc0-11eb-8149-0f01341c0d3a.png)

**Comment by thoughtlessnerd** [comment, marked as answer]
There's no way to delete solutions yet.
